gspread/worksheet.py

```diff
@@ -3201,3 +3204,88 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        values: Iterable[Any],
+        *,
+        inputMessage: Optional[str] = None,
+        strict: bool = False,
+        showCustomUi: bool = False,
+    ) -> Any:
+        """Adds a data validation rule to any given range.
+
+        .. note::
+
+            ``condition_type`` values are explained here: `ConditionType`_
+
+            .. _ConditionType: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#ConditionType
+
+
+        :param str source: The A1 notation of the source range to move
+        :param condition_type: The sort of condition to apply.
+        :param values: List of condition values.
+        :type values: Any
+        :param str inputMessage: Message to show for the validation.
+        :param bool strict: Whether to reject invalid data or not.
+        :param bool showCustomUi: Whether to show a custom UI(Dropdown) for list values.
+
+        **Examples**
+
+        .. code-block:: python
+            import gspread
```

Comment:
you do need a blank line here between `.. code-block:: python` and the first line of code bellow in order to pass documentation build step

Reply:
This has not been resolved. Please add a blank line before you start your code lines.